recipes-devtools/hackage/hkg-base64-bytestring_1.0.0.1.bb

```diff
@@ -0,0 +1,10 @@
+DESCRIPTION = "Fast and efficient encoding and decoding of base64-encoded strings."
+LICENSE = "BSD3"
+LIC_FILES_CHKSUM="file://LICENSE;md5=5c600bfc7377e36f0fb87990f96821e5"
+
+inherit hackage
+
+SRC_URI[md5sum] = "bfe70cca4c75f09f02a8299e19421afc"
+SRC_URI[sha256sum] = "ab25abf4b00a2f52b270bc3ed43f1d59f16c8eec9d7dffb14df1e9265b233b50"
+
+inherit hackage
```

Comment:
There is already an `inherit hackage` above.

Reply:
Fixed.  Slipped through.  I need to visit the eye doc :joy: